go/tasks/plugins/presto/utils/input_interpolator.go

```diff
@@ -0,0 +1,137 @@
+package presto
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+	"regexp"
+	"strings"
+
+	"github.com/golang/protobuf/ptypes"
+	pb "github.com/lyft/flyteidl/gen/pb-go/flyteidl/core"
+	"github.com/lyft/flytestdlib/logger"
+)
+
+// Matches any pair of open/close mustaches that contains a variable inside (e.g. `{{ abc }}`)
+var inputMustacheRegex = regexp.MustCompile(`{{\s*[^\s]+\s*}}`)
```

Comment:
Let's not have our own matching and our own format here... please use the one that already exist [here](https://github.com/lyft/flyteplugins/blob/dc076e467ed5fbf34fce3506db676a8556376463/go/tasks/pluginmachinery/utils/template.go#L34).

Reply:
@wild-endeavor and I discussed this last week and came to the conclusion that we can't just use this out-of-the-box as inputs in the Presto task (as well as how interpolation is  done on the query plus routing_group, catalog, and schema) are being handled a little differently